SevenRed/MainForm.cs

```diff
@@ -0,0 +1,198 @@
+            {
+                Left += e.X - lastPoint.X;
+                Top += e.Y - lastPoint.Y;
+            }
+        }
+
+        /// <summary>
+        /// Find point of movement
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void MainForm_MouseDown(object sender, MouseEventArgs e)
+        {
+            lastPoint = new Point(e.X, e.Y);
+        }
+
+        /// <summary>
+        /// Add Cards to TextBoxes and check of correct data
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void AddCards_Click(object sender, EventArgs e)
+        {
+            List<string> whiteDeck = WhiteDeckText.Lines.ToList();
+            List<string> blackDeck = BlackDeckText.Lines.ToList();
+
+            try
+            {
+                if (whiteDeck.Count <= 8 && blackDeck.Count <= 8)
+                {
+                    WhiteDeck = Service.ConvertListStringToListCard(whiteDeck);
+                    BlackDeck = Service.ConvertListStringToListCard(blackDeck);
+                    
+                    List<TextBox> WhiteSlot = new List<TextBox>(7) { whiteCard1, whiteCard2, whiteCard3, whiteCard4, whiteCard5, whiteCard6, whiteCard7 };
+                    List<TextBox> BlackSlot = new List<TextBox>(7) { blackCard1, blackCard2, blackCard3, blackCard4, blackCard5, blackCard6, blackCard7 };
+
+                    ResetTextbox(WhiteSlot);
+                    ResetTextbox(BlackSlot);
+
+                    EnterCardsToWinForms(WhiteDeck, WhiteSlot);
+                    EnterCardsToWinForms(BlackDeck, BlackSlot);
+                }
+                else
+                {
+                    throw new Exception("Too many cards");
+                }
+            }
+            catch (Exception ex)
+            {
+                MessageBox.Show(ex.Message, "Error", MessageBoxButtons.OK, MessageBoxIcon.Error);
+            }            
+        }
+
+        /// <summary>
+        /// Cleans textboxes
+        /// </summary>
+        /// <param name="Slot"></param>
+        private void ResetTextbox(List<TextBox> Slot)
+        {
+            for (int i = 0; i < Slot.Count; i++)
+            {
+                Slot[i].Text = "";
+                Slot[i].BackColor = Color.FromArgb(((int)(((byte)(37)))), ((int)(((byte)(87)))), ((int)(((byte)(50)))));
+            }
+        }
+
+        /// <summary>
+        /// Enter of textbox text and colors
+        /// </summary>
+        /// <param name="Deck"></param>
+        /// <param name="Slot"></param>
+        private void EnterCardsToWinForms(List<Card> Deck, List<TextBox> Slot)
+        {
+            for (int i = 0; i < Deck.Count; i++)
+            {
+                switch (Deck[i].Nominal)
+                {
+                    case 1:
+                        Slot[i].Text = "1";
+                        Slot[i].ForeColor = Color.White;
```

Comment:
Use mapper from enum color type to `Color` structure. It will help you to avoid switch operators